hdfs3/core.py

```diff
@@ -24,7 +24,7 @@ def hdfs_conf(conf_dir=None):
     """ Load HDFS config from default locations. """
     if conf_dir == None:
         confd = os.environ.get('HADOOP_CONF_DIR', os.environ.get('HADOOP_INSTALL',
-                           '') + '/hadoop/conf')
+                           '/etc') + '/hadoop/conf')
```

Comment:
@martindurant Are you sure CDH uses `/hadoop/conf/core-site.xml`?

I have a CDH cluster here that has `/etc/hadoop/conf/core-site.xml`.

And the `/etc` default only kicks in when the two variables are not set